From e38388ac495e1b7c774a25ac7eb1ac74d213b496 Mon Sep 17 00:00:00 2001
From: Ash Charles <ashcharles@gmail.com>
Date: Tue, 27 Aug 2013 15:53:17 -0700
Subject: [PATCH 8/8] OMAP3: overo: add some Gumstix expansion boards

Add EEPROM ID switches for Stagecoach, Thumbo and Turtlecore Gumstix
expansion boards.

Upstream-Status: Pending

Signed-off-by: Ash Charles <ashcharles@gmail.com>
---
 board/overo/overo.c |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/board/overo/overo.c b/board/overo/overo.c
index 397aa70..98ef0b0 100644
--- a/board/overo/overo.c
+++ b/board/overo/overo.c
@@ -57,6 +57,9 @@ DECLARE_GLOBAL_DATA_PTR;
 #define GUMSTIX_PINTO			0x07000200
 #define GUMSTIX_GALLOP43		0x08000200
 #define GUMSTIX_ALTO35			0x09000200
+#define GUMSTIX_STAGECOACH		0x0A000200
+#define GUMSTIX_THUMBO			0x0B000200
+#define GUMSTIX_TURTLECORE		0x0C000200
 
 #define ETTUS_USRP_E			0x01000300
 
@@ -330,6 +333,21 @@ int misc_init_r(void)
 		MUX_ALTO35();
 		setenv("defaultdisplay", "lcd35");
 		break;
+	case GUMSTIX_STAGECOACH:
+		printf("Recognized Stagecoach expansion board (rev %d %s)\n",
+			expansion_config.revision,
+			expansion_config.fab_revision);
+		break;
+	case GUMSTIX_THUMBO:
+		printf("Recognized Thumbo expansion board (rev %d %s)\n",
+			expansion_config.revision,
+			expansion_config.fab_revision);
+		break;
+	case GUMSTIX_TURTLECORE:
+		printf("Recognized Turtlecore expansion board (rev %d %s)\n",
+			expansion_config.revision,
+			expansion_config.fab_revision);
+		break;
 	case ETTUS_USRP_E:
 		printf("Recognized Ettus Research USRP-E (rev %d %s)\n",
 			expansion_config.revision,
-- 
1.7.10.4

